ha/core/controllers/pcs/node_controller.py

```diff
@@ -176,6 +167,77 @@ def status(self, nodeids: list = None) -> dict:
         """
         raise HAUnimplemented("This operation is not implemented.")
 
+    def check_cluster_feasibility(self, nodeid: str) -> dict:
```

Comment:
is this function exposed
if yes use @controller_error_handler

Reply:
Added @controller_error_handler